invenio_accounts/templates/invenio_accounts/settings/security.html

```diff
@@ -24,40 +24,42 @@
 #}
 {%- extends config.ACCOUNTS_SETTINGS_TEMPLATE %}
 
-{% set panel_title = _('Security') %}
-{% set panel_subtitle = _('This is a list of devices that have logged into your account.') %}
+{% set panel_title = _('Sessions') %}
 {% set panel_icon = 'fa fa-shield fa-fw' %}
+
 {% from "invenio_accounts/_macros.html" import render_field, form_errors %}
 
 {% set revoke_action = _("Revoke") %}
-{% set session_title = _("Session identifier") %}
-{% set signed_in_title = _("Signed in") %}
 
 
 {% block settings_form %}
-  <p>{{panel_subtitle}}</p>
+  <p>{{_('This is a list of devices that have logged into your account.')}}</p>
   <hr>
-  {% for session in sessions %}
+  {%- for session in sessions %}
+  {%- set is_current_sid = is_current(session.sid_s) %}
+  {%- set form = formclass(sid_s=session.sid_s) %}
   <div class="row record-elem">
     <div class="col-xs-1">
-      {% if is_current(session.sid_s) %}
-        <i class="fa fa-circle" aria-hidden="true"></i>
-      {% else %}
-        <i class="fa fa-circle-o" aria-hidden="true"></i>
-      {% endif %}
+      {%- if is_current_sid %}
+        <i class="fa fa-circle" style="color: #28a745;" aria-hidden="true"></i>
```

Comment:
Then it would be nice to have a new css class so one can simply override the css instead of whole template.